### DISCUSSION TITLE
Discussions for website is now open!

### DISCUSSION THREAD

**Discussion by Hans5958** [ScratchAddons/website-v2#172] [Announcements]
## 👋 Welcome!

Hey, everyone. I've decided to open the Discussions, primarily for Giscus, but you can use it to discuss everything related to the website, either about the localization process, how the website works, etc. 

**Reminder: Except for the page comments, please do not discuss the Scratch Addons itself here! [The main repository has its own forum!](https://github.com/ScratchAddons/ScratchAddons/discussions/)**

**Comment by CST1229** [comment]
> **please do not discuss about *the* Scratch Addons itself here**

> but you can use it to *discus* everything related to the website

grammar mistakes

> **With the exception of *the* page comments**

> I've decided to open *the* Discussions

possible grammar mistakes

**Reply by Hans5958** [reply]
![image](https://user-images.githubusercontent.com/11584103/158376289-ee174a22-66cf-4fc8-b978-aec6b91a852c.png)

Come on, man!
